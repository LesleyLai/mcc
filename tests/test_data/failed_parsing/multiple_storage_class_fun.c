
/* A function declaration can't have multiple storage class keywords */
static int extern foo(void) {
  return 0;
}

int main(void) {
  return foo();
}
