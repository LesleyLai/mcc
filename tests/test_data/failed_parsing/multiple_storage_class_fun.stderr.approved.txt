{{filename}}:1:12: Error: multiple storage classes in declaration specifiers
1 | static int extern foo(void) {
  |            ^~~~~~

